csrankings-t.csv

```diff
@@ -719,6 +719,7 @@ Tomer Koren,Tel Aviv University,https://tomerkoren.github.io,wGG1voYAAAAJ
 Tomer Michaeli,Technion,https://tomer.net.technion.ac.il,n2EbR2cAAAAJ
 Tomer Shlomi,Technion,http://www.cs.technion.ac.il/~tomersh,NOSCHOLARPAGE
 Tomi Männistö,University of Helsinki,https://www.cs.helsinki.fi/u/tomimann,ZSX8_ToAAAAJ
+Tommaso Caselli,University of Groningen,https://www.researchgate.net/profile/Tommaso-Caselli,fxQvP_QAAAAJ

```

Comment:
Not clearly in CS (see inclusion criteria).

Reply:
linked to RUG webpage that shows CS